awx/main/managers.py

```diff
@@ -115,7 +116,7 @@ def me(self):
 
     def register(self, uuid=None, hostname=None, ip_address=None, node_type='hybrid', defaults=None):
         if not uuid:
-            uuid = settings.SYSTEM_UUID
```

Comment:
oh, wait, we said we wanted to just let this field resolve to the default value if not specified

Reply:
alas, it doesn't have a default, will add that

Reply:
done